packages/kbn-optimizer/src/worker/run_compilers.ts

```diff
@@ -111,6 +116,15 @@ const observeCompiler = (
 
           if (!parsedPath.dirs.includes('node_modules')) {
             referencedFiles.add(path);
+
+            if (path.endsWith('.scss')) {
+              workUnits += 100;
```

Comment:
Curious how you came up with `100`.

Reply:
Just a general idea that scss accounts for about 40% of our compilation time, and there are "thousands" of files... and I wanted the sass bundles to end up on different workers, so I just tried 100 and it seemed to have enough of an impact on build times on my iMac. Ultimately it's not going to make a difference for just about anyone but me so I didn't think too hard about it.

Reply:
[`f7d9386` (#70911)](https://github.com/elastic/kibana/pull/70911/commits/f7d9386d9bb10f7e30d10ca9bcc6b930ca5dd63f)